### PR TITLE
PKGBUILDs for gregorio for the Arch User Repository

### PR DESCRIPTION
**Pull request by avbop**

### CODE REVIEW

File: gregorio-git/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  ./configure --prefix=/usr || return 1
+}
+
+build() {
+  msg "Compiling gregorio..."
+  cd "$srcdir/$pkgbase"
+  make -j || return 1
+  msg "Building fonts..."
+  cd "$srcdir/$pkgbase/fonts"
+  make -j fonts || return 1
```
Comment (avbop) [inline gregorio-git/PKGBUILD:39]: This works in the latest develop branch (fontforge -script).

**Comment by eroux** [comment]
These files won't move a lot, I cannot test them, and you did a very good job, so I give you write access to the repo

File: gregorio/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+pkgname=$pkgbase
+pkgver=2.4.2
+pkgrel=1
+pkgdesc="Command-line tool to typeset Gregorian chant"
+url=http://home.gna.org/gregorio
+arch=("i686" "x86_64")
+license=("GPL")
+depends=("texlive-core" "texlive-fontsextra" "texlive-bin" "texlive-formatsextra" "texlive-latexextra")
+conflicts=("gregorio-svn" "gregorio-git" "gregoriotex")
+provides=("gregorio-git")
```
Comment (eschwab) [inline gregorio/PKGBUILD:14]: Oops, you added `-git` to the stable PKGBUILD and not the git build.

Comment (avbop) [inline gregorio/PKGBUILD:14]: Fixed, thanks. I looked at some other examples, and I think they should both be `provides=("gregorio")`; it looks like it's generally used to make it clear that `foo` and `foo-git` provide the same package.

**Comment by avbop** [comment]
@eroux Thank you! I'm going to test these one more time and then upload them to the AUR.

**Pull request merged**